lib/Watcher.php

```diff
@@ -116,6 +117,14 @@ public function postWrite(Node $node) {
 			return;
 		}
 
+		// NOTE: The hooks also report the creation of thumbnails, and other files.
+		// The way to differentiate between them, is that the user files starting with the userId, and the thumbains as appdata_ocs######
+		$absPath = ltrim($node->getPath(), '/');
+		$owner = explode('/', $absPath)[0];
```

Comment:
concept of "owner" exist for local files only. OK, maybe for shared? But for external files, it does not make sense, I think? Dunno, never studied that

Reply:
I do not know, but look at my comments that until recently were pending ....

Reply:
ok, got it - so, you basically detect storage type, based on `getPath()` ?:) This miiiight work, if we do not have other options:)